app/models/recurring_bill.rb

```diff
@@ -7,5 +7,5 @@ class RecurringBill < ApplicationRecord
   validates :due_day, presence: true, numericality: { greater_than: 0, lower_than: 31 }
   validates :group_id, presence: true
```

Comment:
Hey, we have a duplicate line here:
https://github.com/wfbraga/finco/blob/a8acc0974946d40cebd5b987109866c2caca03a7/app/models/recurring_bill.rb#L5-L6

Also, I think we can remove this line because `belongs_to :group` is already doing the job:
```suggestion
```